examples/random_module.py

```diff
@@ -11,6 +11,9 @@
 bad = random.uniform()
 bad = random.triangular()
 bad = random.randbytes()
+bad = random.sample()
```

Comment:
These changes will require an update to the functional tests to update the count of the number of fail checks.

Reply:
Done.